test/internal/opts/process_compiler_opts_tests.bzl

```diff
@@ -149,8 +149,6 @@ def process_compiler_opts_test_suite(name):
             "arm64-apple-ios15.0-simulator",
             "-sdk",
             "__BAZEL_XCODE_SDKROOT__",
-            "-F__BAZEL_XCODE_DEVELOPER_DIR__/Platforms/iPhoneSimulator.platform/Developer/Library/Frameworks",
-            "-F__BAZEL_XCODE_SDKROOT__/Developer/Library/Frameworks",
```

Comment:
Can you add `-F` to the test that is testing `-F`? In all the same variations?

Reply:
for sure, updated!